### PR TITLE
update: Add more Simplex Chat info

### LINKED ISSUE #2776: SimpleX new audit + add more description
### Affected page

https://www.privacyguides.org/en/real-time-communication/#simplex-chat

### Description

https://simplex.chat/blog/20241014-simplex-network-v6-1-security-review-better-calls-user-experience.html (new audit)

The current description for SimpleX Chat sounds like any other messenger that does nothing innovative other than requiring no user credentials. We can add features mentioned in "What makes SimpleX private" on their [website](https://simplex.chat/)

### Sources

https://simplex.chat/blog

### Before submitting

- [X] I am reporting something that is verifiably incorrect, not a suggestion or opinion.
- [X] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).

### PR DESCRIPTION
**Pull request by friadev**
List of changes proposed in this PR:

- fixed stylistic inconsistency 
- added new July 2024 audit
- added more detailed info from the simplex website

closes #2776

### CODE REVIEW

File: docs/real-time-communication.md [hunk 1]
```diff
@@ -133,9 +133,9 @@ Molly and Molly-FOSS support [reproducible builds](https://github.com/mollyim/mo
 
 </div>
 
-SimpleX Chat [was audited](https://simplex.chat/blog/20221108-simplex-chat-v4.2-security-audit-new-website.html) by Trail of Bits in October 2022.
+SimpleX Chat [was audited](https://simplex.chat/blog/20241014-simplex-network-v6-1-security-review-better-calls-user-experience.html#simplex-cryptographic-design-review-by-trail-of-bits) by Trail of Bits in July 2024 [and previously](https://simplex.chat/blog/20221108-simplex-chat-v4.2-security-audit-new-website.html) by Trail of Bits in October 2022.
 
-SimpleX Chat supports basic group chatting functionality, direct messaging, and editing of messages and markdown. E2EE Audio and Video calls are also supported. Your data can be exported and imported onto another device, as there are no central servers where this is backed up.
+SimpleX Chat supports group chatting functionality, direct messaging, and editing of messages and markdown. E2EE Audio and Video calls are also supported. Your data can be exported and imported onto another device, as there are no central servers where this is backed up.
```
Comment (redoomed1) [inline docs/real-time-communication.md:138]:
```suggestion
```
The content here was moved to other sections of the SimpleX listing.

**Review by redoomed1 — CHANGES_REQUESTED** [review]
The changes in this PR make the card description for SimpleX a little too long IMO. My suggestions below attempt to spread the added info evenly throughout the listing.

The info about Tor and message padding was replaced with the link to the SimpleX repo README in line 138, which mentions those features.

File: docs/real-time-communication.md [hunk 2]
```diff
@@ -133,9 +133,9 @@ Molly and Molly-FOSS support [reproducible builds](https://github.com/mollyim/mo
 
 </div>
 
-SimpleX Chat [was audited](https://simplex.chat/blog/20221108-simplex-chat-v4.2-security-audit-new-website.html) by Trail of Bits in October 2022.
+SimpleX Chat [was audited](https://simplex.chat/blog/20241014-simplex-network-v6-1-security-review-better-calls-user-experience.html#simplex-cryptographic-design-review-by-trail-of-bits) by Trail of Bits in July 2024 [and previously](https://simplex.chat/blog/20221108-simplex-chat-v4.2-security-audit-new-website.html) by Trail of Bits in October 2022.
```
[the remark this answers is not in the document]
Comment (kimg45) [reply docs/real-time-communication.md:138]: I think the most recent audit should be listed first.

**Review by redoomed1 — APPROVED** [review]
LGTM

File: docs/real-time-communication.md [hunk 3]
```diff
@@ -112,11 +112,13 @@ Molly and Molly-FOSS support [reproducible builds](https://github.com/mollyim/mo
...
 
-**SimpleX** Chat is an instant messenger that doesn't depend on any unique identifiers such as phone numbers or usernames. Its decentralized network makes SimpleX Chat an effective tool against [:material-close-outline: Censorship](basics/common-threats.md#avoiding-censorship){ .pg-blue-gray }. Users of SimpleX Chat can scan a QR code or click an invite link to participate in group conversations.
+**SimpleX Chat** is an instant messenger that doesn't depend on any unique identifiers such as phone numbers or usernames. Its decentralized network makes SimpleX Chat an effective tool against [:material-close-outline: Censorship](basics/common-threats.md#avoiding-censorship){ .pg-blue-gray }.
+
+The app provides direct messaging, group chats, and E2EE calls secured with the [SimpleX Messaging Protocol](https://github.com/simplex-chat/simplexmq/blob/stable/protocol/simplex-messaging.md), which uses double ratchet encryption with quantum resistance. Additionally, SimpleX Chat provides metadata protection by using unidirectional ["simplex queues"](https://github.com/simplex-chat/simplexmq/blob/stable/protocol/simplex-messaging.md#simplex-queue) to deliver messages.
```
Comment (jonaharagon) [inline docs/real-time-communication.md:117]:
```suggestion
```

see below

File: docs/real-time-communication.md [hunk 4]
```diff
@@ -133,9 +135,12 @@ Molly and Molly-FOSS support [reproducible builds](https://github.com/mollyim/mo
 
 </div>
```
Comment (jonaharagon) [inline docs/real-time-communication.md:137]:
```suggestion

SimpleX provides direct messaging, group chats, and E2EE calls secured with the [SimpleX Messaging Protocol](https://github.com/simplex-chat/simplexmq/blob/stable/protocol/simplex-messaging.md), which uses double ratchet encryption with quantum resistance. Additionally, SimpleX Chat provides metadata protection by using unidirectional ["simplex queues"](https://github.com/simplex-chat/simplexmq/blob/stable/protocol/simplex-messaging.md#simplex-queue) to deliver messages.

```

stylistically I think the recommendation boxes shouldn't be overloaded with content (_especially_ technical content like what encryption protocol an app uses)

**Review by jonaharagon — APPROVED** [review]

**Pull request merged**